File: Practice/src/introduction_coding_tests/GetDiscountAtClothingStore.js
function solution(price) {
    var n = 10 ** 5;
    if (price >= 5 * n) return parseInt(price * 0.8);
    else if (price >= 3 * n) return parseInt(price * 0.9);
    else if (price >= n) return parseInt(price * 0.95);
    return price;
}

File: Practice/src/introduction_coding_tests/GetDiscountAtClothingStore.py
def solution(price):
    n = 10 ** 5
    if price >= n * 5:
        return int(price * 0.8)
    elif price >= n * 3:
        return int(price * 0.9)
    elif price >= n:
        return int(price * 0.95)
    return price
